src/sentry/static/sentry/app/views/releases/list/releaseCard.tsx

```diff
@@ -40,191 +30,84 @@ const ReleaseCard = ({
-              </ColumnTitle>
-            </CommitsColumn>
-
-            <NewIssuesColumn>
-              <ColumnTitle>{t('New issues')}</ColumnTitle>
-            </NewIssuesColumn>
-          </HeaderLayout>
-
-          <Layout>
-            <VersionColumn>
-              <VersionWrapper>
-                <Version version={version} tooltipRawVersion truncate anchor={false} />
-              </VersionWrapper>
-            </VersionColumn>
-
-            <CreatedColumn>
-              <TextOverflow>
-                {lastDeploy?.dateFinished && <StyledDeployBadge deploy={lastDeploy} />}
-                <TimeSince date={lastDeploy?.dateFinished || dateCreated} />
-              </TextOverflow>
-            </CreatedColumn>
-
-            <CommitsColumn>
-              <CommitsWrapper>
-                {commitCount > 0 ? (
-                  <ReleaseStats release={release} withHeading={false} />
-                ) : (
-                  <NotAvailable />
-                )}
-              </CommitsWrapper>
-            </CommitsColumn>
-
-            <NewIssuesColumn>
-              <Feature features={['global-views']}>
-                {({hasFeature}) =>
-                  hasFeature ? (
-                    <Tooltip title={t('Open in Issues')}>
-                      <Link to={getReleaseNewIssuesUrl(orgSlug, null, version)}>
-                        <Count value={release.newGroups || 0} />
-                      </Link>
-                    </Tooltip>
-                  ) : (
-                    <Count value={release.newGroups || 0} />
-                  )
-                }
-              </Feature>
-            </NewIssuesColumn>
-          </Layout>
-        </StyledPanelItem>
-      </PanelBody>
-
-      <ReleaseHealth
-        release={release}
-        orgSlug={orgSlug}
-        location={location}
-        showPlaceholders={showHealthPlaceholders}
-        selection={selection}
-      />
-    </StyledPanel>
+    <StyledReleaseCard reloading={reloading ? 1 : 0}>
+      <ReleaseInfo>
+        <ReleaseInfoHeader>
+          <Version version={version} tooltipRawVersion truncate anchor={false} />
+          {commitCount > 0 ? (
+            <ReleaseStats release={release} withHeading={false} />
+          ) : (
+            <NotAvailable />
+          )}
+        </ReleaseInfoHeader>
+        <ReleaseInfoSubheader>
+          {lastDeploy?.dateFinished && <StyledDeployBadge deploy={lastDeploy} />}
+          <TimeSince date={lastDeploy?.dateFinished || dateCreated} />
+        </ReleaseInfoSubheader>
+      </ReleaseInfo>
+
+      <ReleaseProjects>
+        <ReleaseHealth
+          release={release}
+          orgSlug={orgSlug}
+          location={location}
+          showPlaceholders={showHealthPlaceholders}
+          selection={selection}
+        />
+      </ReleaseProjects>
+    </StyledReleaseCard>
   );
 };
 
-const StyledPanel = styled(Panel)<{reloading: number}>`
+const StyledReleaseCard = styled(Panel)<{reloading: number}>`
```

Comment:
as we are styling the Panel component, I would keep the name `StyledPanel` or if necessary I would only name it `ReleaseCard`